recipes/linux-headers/linux-headers.inc

```diff
@@ -4,7 +4,7 @@ inherit kernel-headers
 RECIPE_TYPES = "machine sdk"
 
 require conf/fetch/kernelorg.conf
-SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${KERNEL_VERSION_PATCHLEVEL}/linux-${PV}.tar.bz2"
+SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${@'${PV}'.split('.')[0]+'.x'}/linux-${PV}.tar.xz"
```

Comment:
Totally agree.  I bluntly missed the context (.inc file and not class).  As far as I am concerned, .inc files are for in-tree use only.